### PR TITLE
Add ANZ/NAB/ING Australian banks

### PR DESCRIPTION
**Pull request by jdgordon**
Adds support for 3 Australian bank exports.

* ANZ (via ANZ.csv)
* ING (via Transactions.csv)
* NAB (via TransactionHistory.csv)

### CODE REVIEW

**Comment by jdgordon** [comment]
@nocalla 

> Thanks for the contribution. It might make more sense to use Payee instead of Memo in these fields. Memo is automatically filled in with the Payee details by the script and then Payee details can be used by YNAB directly for renaming purposes.
> 

"V2718 16/04 GLICKS CAKES/BAGELS PL BALACLAVA " is an example of one of those memo lines, they have way more information (and are mostly not static) that I'd expect to put in the payee field, happy for you to change it to payeeif you feel thats more inline with the other importers though.



> Does the double Inflow column work properly if there's numbers in both columns of the same row?

I cant imagine ever having both values set, and presumably the import code would just take the second field. (cant test now unfortunately)

File: bank2ynab.conf [hunk 1]
```diff
@@ -82,6 +82,31 @@ Source CSV Delimiter = ;
 Header Rows = 0
 Input Columns = skip,Date,Memo,skip,skip,skip,Inflow
 
+[AU ANZ]
+# Transactions.csv
+Source Filename Pattern = ANZ
+Header Rows = 0
+Input Columns = Date,Inflow,Memo
+Date Format = %d/%m/%y
```
Comment (torbengb) [inline bank2ynab.conf]: Just checking: do these formats really have 2-digit years? 
(4-digit would be uppercase %Y)

**Comment by nocalla** [comment]
I think that would be more in line with the functionality of the other formats, as it allows for automatic conversion by YNAB of a Payee name containing, say, GLICKS CAKES, to a payee called Glicks Bakery. So the extra info will be culled if it's a repeat Payee or can just be ignored and deleted. The script relies on the presence of a Payee field to some extent and this method preserves the info in the Memo field without losing any info. 

The logic of the multiple input columns is to accommodate multiple text fields, e.g. memos spread across a few cells. The values all get merged into one, which I don't think would work well with numbers! What's the logic of the 2 inflow columns? Is it at all possible that the two could get filled together? If not, then that's fine.

**Comment by torbengb** [comment]
Regarding memo-vs-payee, I have learned that YNAB's payee management is very powerful. My own bank doesn't provide payee information but the memo can be used. 

... aaand I saw that Nocalla has posted exactly this while I am typing, so I'll just stop right here :-)

**Comment by nocalla** [comment]
Merging this. We can amend the year if we get more feedback.

**Pull request merged**